eth/tracers/api.go

```diff
@@ -769,7 +769,7 @@ func (api *API) standardTraceBlockToFile(ctx context.Context, block *types.Block
 		// Execute the transaction and flush any traces to disk
 		vmenv := vm.NewEVM(vmctx, txContext, statedb, chainConfig, vmConf)
 		statedb.SetTxContext(tx.Hash(), i)
-		l1DataFee, err := fees.CalculateL1DataFee(tx, statedb, api.backend.ChainConfig(), block.Number())
+		l1DataFee, err := fees.CalculateL1DataFee(tx, statedb, chainConfig, block.Number())
```

Comment:
It looks like the only override that we might apply is `BerlinBlock`, is that intended?

Reply:
1. that's the only one override i find in this file
2. the overridden code is from 3-year-ago upstream, not made by us.
3. in line 770, the test code use `chainConfig`, but line 772 is introduced by use and it's using `api.backend.ChainConfig()`

Reply:
Should we make it more generic? https://github.com/ethereum/go-ethereum/blob/master/eth/tracers/api.go#L761-L769 Or is that a lot of extra work?

Reply:
yes on "upstream" branch it's more generic, we can also update here on `develop` branch, I can update